omas/machine_mappings/d3d.py

```diff
@@ -1022,6 +1061,23 @@ def magnetics_probes_data(ods, pulse):
             data_norm=1.0,
         )
 
+    
+    for compfile in ['btcomp','ccomp','icomp']
+        comp = get_support_file(OMFITd3dcompfile, support_filenames('d3d', compfile, pulse))
+        for shot in comp:
+            if pulse > compshot:
+                compshot = shot
+        for compsig in comp[compshot]:
+            compsig_data = ptdata2(f"{compsig}",{pulse})
+            compsig_time = dim_of(ptdata2(f"{compsig}",{pulse}),0)/1000
```

Comment:
@jmcclena I don't see a `ptdata2` function as part of this PR

Reply:
`ptdata2` is a server side tdi function.

Reply:
Thanks. It was untested and poor guessing on my part.  Fixed in [26f2220](https://github.com/gafusion/omas/pull/193/commits/26f2220ff0f541b673d20a4e75105116aba48723)